e2e/main_test.go

```diff
@@ -72,6 +73,8 @@ func TestMain(t *testing.T) {
 
 	defer testContext.Cleanup()
 
+	suite.Run(t, &SanitySuite{testContext: testContext})
```

Comment:
Don't use sanity, we just cleanup this term from ramen recently (#693).

Maybe call it BasicSuite? we can have ExtendedSuite later of maybe several suites 
focusing on specific configurations.